roles/cockpit-ui/tasks/main.yml

```diff
@@ -0,0 +1,22 @@
+---
+- name: Expose registry with route
+  command: oc expose service docker-registry
+
+- name: Install Cockpit template
+  command: oc create -f registry-console.yaml -n default
```

Comment:
Remove this step if example templates are already installed via previous playbook.